tests/integration/test_api.py

```diff
@@ -54,3 +54,27 @@ def test_remove_snap(test_snap):
     response = wait_for(snap_http.remove)("test-snap")
     assert response.status_code == 202
     assert is_snap_installed("test-snap") is False
+
+
+def test_sideload_snap(local_test_snap_path):
```

Comment:
please add a test with more than a single `file_path` so we get the n > 1 case covered.

I guess that means you need to have two test snaps to use? Or can you use the same one twice?